modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-getting-started.adoc
+
+[id='managing-memcached-oeprator-using-olm_{context}']
+= Managing a Memcached Operator using the Operator Lifecycle Manager
+
+The previous section has covered manually running an Operator. In the next
+sections, we will explore using the Operator Lifecycle Manager (OLM), which is
+what enables a more robust deployment model for Operators being run in
+production environments.
+
+The OLM helps you to install, update, and generally manage the lifecycle of all
+of the Operators (and their associated services) on a Kubernetes cluster. It
+runs as an Kubernetes extension and lets you use `kubectl` for all the lifecycle
+management functions without any additional tools.
+
+.Prerequisites
+
+- OLM installed on a Kubernetes-based cluster (v1.8 or above to support the
+`apps/v1beta2` API group), for example {product-title} 3.11 with Technology
+Preview OLM enabled
+- Memcached Operator built
+
+.Procedure
+
+. *Generate an Operator manifest.*
++
+An Operator manifest describes how to display, create, and manage the
+application, in this case Memcached, as a whole. It is defined by a
+`CustomServiceVersion` (CSV) object and is required for the OLM to function.
++
+For the purpose of this guide, we will continue with this
+link:https://github.com/operator-framework/getting-started/blob/master/memcachedoperator.0.0.1.csv.yaml[predefined manifest file]
+for the next steps. You can alter the image field within this manifest to
+reflect the image you built in previous steps, but it is unnecessary. In the
+future, the Operator SDK CLI will generate an Operator manifest for you, a
+feature that is planned for the next release of the Operator SDK.
++
+[NOTE]
+====
+See
+link:https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md[Building a CSV for the Operator Framework]
+for more information on manually defining a manifest file.
+====
+
+. *Deploy the Operator.*
+
+.. Deploying an Operator is as simple as applying the Operator’s manifest to the
+desired namespace in the cluster.
++
+----
+$ curl -Lo memcachedoperator.0.0.1.csv.yaml https://raw.githubusercontent.com/operator-framework/getting-started/master/memcachedoperator.0.0.1.csv.yaml
+$ kubectl apply -f memcachedoperator.0.0.1.csv.yaml
+$ kubectl get ClusterServiceVersion-v1s memcachedoperator.v0.0.1 -o json | jq '.status'
+----
+
+.. After applying this manifest, nothing has happened yet, because the cluster does
+not meet the requirements specified in our manifest. Create the
+`CustomResourceDefinition` and RBAC rules for the Memcached type managed by the
+Operator:
++
+----
+$ kubectl apply -f deploy/rbac.yaml
+$ kubectl apply -f deploy/operator.yaml
```

Comment:
Here should generate the CRD, not the operator. Already have a PR for this:  https://github.com/operator-framework/getting-started/pull/23